docs/source/starters/index.md

```diff
@@ -0,0 +1,44 @@
+# New project tools
```

Comment:
It's weird that `/starters/` is for "new project tools", `/starters/starters.html` is for "starters", and `/starters/new_project_tools.html` is for "configure a new project" (looking at the H1 of each page).

Reply:
And speaking specifically about this title, "new project tools" could be read as "tools for new projects" or "new thing called project tools", which was confusing to me at first. I bet we don't want to use the second meaning, given that the "new" tools will stop being new at some point...

Reply:
I think we should fix the naming in the [upcoming PR](https://github.com/kedro-org/kedro/pull/3331) from @AhdraMeraliQB which explicitly writes the content. I can't really guess at what to call the page right now, so that's a placeholder.

You're correct about the `/starters` being an odd place for those tools but it's awkward to name the folder. Maybe "`/project_creation`